src/main/msp/msp.c

```diff
@@ -1130,24 +1130,10 @@ static bool mspProcessOutCommand(mspDescriptor_t srcDesc, int16_t cmdMSP, sbuf_t
 
     case MSP_RAW_IMU:
         {
-#if defined(USE_ACC)
-            // Hack scale due to choice of units for sensor data in multiwii
-
-            uint8_t scale;
-            if (acc.dev.acc_1G > 512 * 4) {
-                scale = 8;
-            } else if (acc.dev.acc_1G > 512 * 2) {
-                scale = 4;
-            } else if (acc.dev.acc_1G >= 512) {
-                scale = 2;
-            } else {
-                scale = 1;
-            }
-#endif
 
             for (int i = 0; i < 3; i++) {
 #if defined(USE_ACC)
-                sbufWriteU16(dst, lrintf(acc.accADC[i] / scale));
+                sbufWriteU16(dst, lrintf(acc.accADC[i]));
```

Comment:
The `scale` code is very suspicious (acc_1G == 512 is scaled to 256, but acc_1G == 1024 is scaled to 512). 
It seems that 512 * 4 and 512 * 8 is quite common, and they are sent unscaled. 

Sending value unscaled (as in this PR) is probably best option. 
And possibly add some mechanism to get `acc_1G` over MSP, so client can get correct scaling.

Reply:
okay, i'm following now.   no one gonna `multiwii` anymore -- acc_1G is legacy

Reply:
http://www.multiwii.com/forum/viewtopic.php?t=4284

Reply:
nice, same conclusion a decade ago.